flow/raster_cache.h

```diff
@@ -148,12 +148,15 @@ class RasterCache {
   // if the item was disabled due to conditions discovered during |Preroll|
   // or if the attempt to populate the entry failed due to bounds overflow
   // conditions.
-  // If |is_root_canvas| is false, the raster cache will preserve the original
-  // RTree of cached content.
+  // If |preserve_rtree| is true, the raster cache will preserve the original
+  // RTree of cached content by blitting individual rectangles from the cached
+  // image of the canvas according to the original layer R-Tree (if present).
```

Comment:
"of the canvas" doesn't seem necessary and I'm not sure it really describes the DL cache items very well...?

Reply:
That should have been *from the cached image **to** the canvas*. Canvas referring to the destination canvas.